Apidaze.SDK.Tests.Integration/IvrExample/Program.cs

```diff
@@ -0,0 +1,335 @@
+                        {
+                            LangEnum = LangEnum.JAPANESE,
+                            Text = "そして日本人ですら"
+                        }).AddNode(Wait.SetDuration(1)).AddNode(new Speak()
+                        {
+                            Text =
+                                "You can see our documentation for a full list of supported languages and voices for them.  We'll take you back to the menu for now."
+                        }).AddNode(Wait.SetDuration(2))
+                        .ToXml();
+
+                    handled = WriteResponseStream(response, step1);
+                    break;
+            }
+
+            return handled;
+        }
+
+        private static bool GetStep2(HttpListenerContext context, HttpListenerResponse response, bool handled)
+        {
+            switch (context.Request.HttpMethod)
+            {
+                case "GET":
+                    //Get the current settings
+                    response.ContentType = "text/xml";
+
+                    var step2 = ApidazeScript.Build()
+                        .AddNode(Speak.WithText("You will now be joined to an echo line."))
+                        .AddNode(Echo.SetDuration(500))
+                        .ToXml();
+
+                    handled = WriteResponseStream(response, step2);
+                    break;
+            }
+
+            return handled;
+        }
+
+        private static void ReturnExceptionResponse(HttpListenerResponse response, Exception e)
+        {
+            response.StatusCode = 500;
+            response.ContentType = "text/plain";
+            var buffer = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(e));
+            response.ContentLength64 = buffer.Length;
+            response.OutputStream.Write(buffer, 0, buffer.Length);
+        }
+        private static bool GetStep3(HttpListenerContext context, HttpListenerResponse response, bool handled)
+        {
+            switch (context.Request.HttpMethod)
+            {
+                case "GET":
+                    //Get the current settings
+                    response.ContentType = "text/xml";
+
+                    var step3 = ApidazeScript.Build()
+                        .AddNode(Speak.WithText("You will be entered into a conference call now.  You can initiate more calls to join participants or hangup to leave"))
+                        .AddNode(new Conference() { Name = "SDKTestConference" })
+                        .ToXml();
+
+                    handled = WriteResponseStream(response, step3);
+                    break;
+            }
+
+            return handled;
+        }
+        private static bool GetIntroWav(HttpListenerContext context, HttpListenerResponse response, bool handled)
+        {
+            switch (context.Request.HttpMethod)
+            {
+                case "HEAD":
+                    handled = true;
+                    response.ContentLength64 = 0;
```

Comment:
Oh, that's curious, but if it works than kudos for avoiding implementation :)

Reply:
It works perfect. :)